libs/clients/driving-license/src/lib/utils.ts

```diff
@@ -0,0 +1,28 @@
+import fetch from 'isomorphic-fetch'
```

Comment:
There is a more advanced library for that https://docs.devland.is/libs/clients/middlewares. Unless you really want the success response to be log, it's probably better to use this one

Reply:
i added that one, btw, instead